flink-connector-pulsar/src/test/java/org/apache/flink/connector/pulsar/testutils/runtime/remote/PulsarRemoteRuntime.java

```diff
@@ -28,8 +28,12 @@ public PulsarRemoteRuntime(String serviceUrl, String adminUrl) {
 
     @Override
     public PulsarRuntime withConfigs(Map<String, String> configs) {
-        throw new UnsupportedOperationException(
-                "We can't change the broker configs on a running instance.");
+        if (!configs.isEmpty()) {
```

Comment:
What is the purpose of this change? In what scenario will we pass in an empty Map.

Reply:
This is a local debug class for investigate the root cause from Pulsar side. We may use it with `PulsarTestSuiteBase`, it will pass an empty map by default.

Reply:
Ok